cpp/src/binaryop/jit/code/kernel.cpp

```diff
@@ -31,7 +31,10 @@ R"***(
     template <typename TypeOut, typename TypeLhs, typename TypeRhs, typename TypeOpe>
     __global__
     void kernel_v_s(gdf_size_type size,
-                    TypeOut* out_data, TypeLhs* lhs_data, gdf_data rhs_data) {
+                    TypeOut* out_data,
+                    TypeLhs* lhs_data,
+                    gdf_valid_type* lhs_valid,
```

Comment:
We're moving away from gdf_valid_type (8-bit) to `bit_mask_t` (32-bit). Use it here, and reinterpret_cast from gdf_valid_type  to bitmask_t on the host before calling the kernel.

Reply:
Will do. Thank you for your patience. This is the first time I'm touching the cpp codebase. I'd like to get this right, but I will definitely get it wrong. *Please* nit pick my cpp. I haven't done cpp in a very long time.